doc/man7/property.pod

```diff
@@ -126,16 +126,13 @@ Two levels of property query are supported.
 A context based property query that applies to all fetch operations and a local
 property query.
 Where both the context and local queries include a clause with the same name,
-the local clause is used and the context one ignored.
-For example, a context property query of "fips=yes" and a local property query
-of "fips=no" would result in algorithms that have the "fips" property set t
-"no".
+the local clause overrides the context clause.
 
 =head2 Override
```

Comment:
The term "Override" does not fit anymore, use "Removal" or "Property Removal"  instead?

Or maybe just remove the `=head2 Override` heading, making lines 136ff a second paragraph under `=head2 Global and Local`?

Reply:
I've taken the second option.